migrations/2024-11-10-134311_smoosh-tables-together/up.sql

```diff
@@ -141,60 +117,31 @@ FROM
 ON CONFLICT (person_id,
     target_id)
     DO UPDATE SET
-        person_id = excluded.person_id,
-        target_id = excluded.target_id,
         blocked = excluded.blocked;
 
-INSERT INTO post_actions (person_id, post_id, read_comments, read_comments_amount)
+INSERT INTO post_actions (person_id, post_id, read_comments, read_comments_amount, hidden, liked, like_score, saved)
 SELECT
     person_id,
     post_id,
-    published,
-    read_comments
+    person_post_aggregates.published,
+    person_post_aggregates.read_comments,
+    post_hide.published,
+    post_like.published,
+    post_like.score,
+    post_saved.published
 FROM
     person_post_aggregates
+    FULL JOIN post_hide USING (person_id, post_id)
+    FULL JOIN post_like USING (person_id, post_id)
```

Comment:
Theses `full join` changes to `smoosh` massively increased the time... they're going on 15+ hours now so I just canceled them.

So probably best to revert these to what it was before.

Reply:
I tried a different way. If it's still slower, then I will probably keep only a few changes.

Reply:
Mmmk. I'll test it again this weekend, because it does take a really long time.